src/PatchCollector.php

```diff
@@ -143,7 +143,12 @@ protected function resolveExplicitList(PackageInterface $package, $patches)
         if (!is_array($patches)) {
             $this->handleException(Exception\InvalidPackageConfigurationValue($package, 'extra.patches', $patches, 'The extra.patches configuration must be an array.'));
         } else {
-            $packageDirectory = $this->installationManager->getInstallPath($package);
+            // If the package is the RootPackage, we won't be able to get its install path from the installationmanager
+            if ($package instanceof RootPackageInterface) {
+                $packageDirectory = getcwd();
```

Comment:
I wasn't able to find a better way to resolve the path of the RootPackage instance.